### PR TITLE
(BuZZy1337) escape HTML Tags from selectID.js

### PR DESCRIPTION
**Pull request by BuZZy1337**
As requested by Bluefox ( https://github.com/ioBroker/ioBroker.admin/pull/259#issuecomment-385342717 )

### CODE REVIEW

**Comment by BuZZy1337** [comment]
@GermanBluefox please review before merging. Thanks! :relaxed:

**Comment by BuZZy1337** [comment]
image
.. :+1:

**Comment by BuZZy1337** [comment]
Please do not merge yet!!

**Comment by BuZZy1337** [comment]
Problem by that commit
before:
image

after:
image

States/IDs are missing.. I have to investigate that.

**Comment by BuZZy1337** [comment]
Quick code modification during Coffee break...

Escape HTML now works as expected! :smile:

**Comment by BuZZy1337** [comment]
.. and also implemented https://github.com/ioBroker/ioBroker.admin/issues/256

image
image
image

@GermanBluefox is this okay for you?

File: src/js/admin.js [hunk 1]
```diff
@@ -1587,7 +1587,7 @@ $(document).ready(function () {
                 } else {
                     $('#host-adapters-btn').css('opacity', 0.3);
                 }
-
+                document.title = 'ioBroker.' + tab;
```
[the remark this answers is not in the document]
Comment (BuZZy1337) [reply src/js/admin.js]: Done. :relaxed:

**Comment by GermanBluefox** [comment]
Thank you

**Comment by AlCalzone** [comment]
Shouldn't this be translated like the Tab headers? Currently I get lowercase english names in the title bar but my ioBroker is set to german.

**Comment by BuZZy1337** [comment]
You are right. I will take care about that later. :relaxed:

**Pull request merged**